pkgs/applications/networking/cluster/k3s/builder.nix

```diff
@@ -331,6 +332,17 @@ let
       ldflags = versionldflags;
     }).overrideAttrs
       overrideContainerdAttrs;
+
+  # TODO (#405952): remove this patch. We had to add it to avoid a mass rebuild
+  # for the 25.05 release. Once the above PR is merged, switch back to plain util-linuxMinimal.
+  k3sUtilLinux = util-linuxMinimal.overrideAttrs (prev: {
+    patches =
+      prev.patches or [ ]
+      ++ lib.singleton (fetchpatch {
+        url = "https://github.com/util-linux/util-linux/pull/3479.patch";
```

Comment:
You can use `https://github.com/util-linux/util-linux/commit/7dbfe31a83f45d5aef2b508697e9511c569ffbc8.patch` instead of the pull request patch as it is already merged.

Reply:
Good point. Missed that. We should never point to a pull request in nixpkgs because the hash can get invalidated easily.

Reply:
Forgot about this change. Will make a follow up PR.

Reply:
https://github.com/NixOS/nixpkgs/pull/408973